Packages/com.unity.inputsystem/InputSystem/API/Input.cs

```diff
@@ -937,6 +1006,46 @@ public static bool IsGamepadButtonUp(GamepadButton input, GamepadSlot gamepadSlo
         {
             throw new NotImplementedException();
         }
+#endif
+        
+        private static GamepadSlot FindSlotForGamepad(Gamepad gamepad)
+        {
+            // TODO implement me
```

Comment:
Waiting for "stable arrays" for this one?

Reply:
Yes, some methods are waiting for stable arrays.

But this one is redundant now, so fixed.